go/ct/spc/specification.go

```diff
@@ -1402,6 +1402,26 @@ func getAllRules() []Rule {
 		},
 	})...)
 
+	// notEmpty
+	rules = append(rules, rulesFor(instruction{
```

Comment:
It seems to me that the main intention of introducing this rule is to up code coverage, but it does not add any new specification coverage. This is against the idea of writing up rules.

It would be better if the rules for EXTCODEHASH would follow the pattern of creating a Cartesian product of influencing parameters, to get the full combinations of empty / warm / revision-ranges covered;

Reply:
cartesian product of relevant variables has been added